crates/db-allocator/src/interval_tree.rs

```diff
@@ -921,13 +921,16 @@ mod tests {
         let _ = Range::new(0u64, std::u64::MAX);
     }
 
+    #[allow(clippy::many_single_char_names)]
```

Comment:
Could you please help to split out this into a dedicated PR? And replace "a,b,c,xx" with longer names?

Reply:
fix in pull request #14